app/src/main/kotlin/com/thundermaps/apilib/android/api/resources/ShapeResource.kt

```diff
@@ -0,0 +1,8 @@
+package com.thundermaps.apilib.android.api.resources
+
+import com.thundermaps.apilib.android.api.requests.models.MapboxFeature
+import com.thundermaps.apilib.android.api.responses.models.Result
+
+interface ShapeResource {
+    suspend fun getShape(mapboxFeature: MapboxFeature): Result<String>
```

Comment:
This gets a collection of features or a Turf.FeatureCollection. It would be nice if the API library decoded the response. As I mentioned the iOS uses Turf for this as its the same as what the Mapbox SDK uses - Looks like they have an Android version too https://docs.mapbox.com/android/java/guides/turf/